storage/storage_test.go

```diff
@@ -1149,17 +1149,22 @@ func TestDBStorageWriteRecommendationForClusterAlreadyStoredAndDeleted(t *testin
 	storage.SetClustersLastChecked(dbStorage, testdata.ClusterName, time.Now())
 
 	expects.ExpectBegin()
+	expects.ExpectQuery(`SELECT created_at FROM recommendation`).
+		WillReturnRows(expects.NewRows([]string{"created_at"}).AddRow(time.Time{})).
+		RowsWillBeClosed()
 	expects.ExpectExec("DELETE FROM recommendation").
 		WillReturnResult(driver.RowsAffected(3))
 	expects.ExpectExec("INSERT INTO recommendation").
 		WillReturnResult(driver.ResultNoRows)
 	expects.ExpectCommit()
+	expects.ExpectClose()
+	expects.ExpectClose()
```

Comment:
why is it necessary to expectClose twice?

Reply:
it will fail if i remove one, and it will fail even if I don't call Close()